### PR TITLE
Add Celoscan (EIP3091) and lowercase shortName

### PR DESCRIPTION
**Pull request by matthewlilley**
Add Celoscan (EIP3091) and lowercase shortName to adhere to EIP-3770

### CODE REVIEW

File: _data/chains/eip155-42220.json [hunk 1]
```diff
@@ -12,8 +12,12 @@
   "rpc": ["https://forno.celo.org", "wss://forno.celo.org/ws"],
   "faucets": ["https://free-online-app.com/faucet-for-eth-evm-chains/"],
   "infoURL": "https://docs.celo.org/",
-
   "explorers": [
+    {
+      "name":"Celoscan",
+      "url":"https://celoscan.io",
+      "standard":"EIP3091"
```
Comment (ligi) [inline _data/chains/eip155-42220.json:19]:
```suggestion
      "name": "Celoscan",
      "url": "https://celoscan.io",
      "standard": "EIP3091"
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Comment by matthewlilley** [comment]
@ligi Thanks. Looks like this one is good to go.

**Review by tazimamatovzeka — APPROVED** [review]

**Review by FrederikBolding — APPROVED** [review]

**Review by tazimamatovzeka — COMMENTED** [review]
Ayzat

**Review by ligi — APPROVED** [review]

**Comment by tazimamatovzeka** [comment]
IE03091

**Pull request merged**